contrib/ci-horizen/dockerfiles/amd64/linux/ubuntu_focal/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN set -euxo pipefail \
     && apt-get -y --no-install-recommends install aria2 autoconf automake bsdmainutils build-essential ca-certificates \
        cmake curl dirmngr fakeroot git g++-multilib gnupg2 help2man libc6-dev libgomp1 libtool lintian m4 ncurses-dev \
        pigz pkg-config pv python3-dev python3-pip python3-setuptools python3-wheel time unzip wget zlib1g-dev \
-    && pip3 install pycryptodome b2 pyblake2 pyzmq websocket-client pyzmq \
```

Comment:
Just seen by chance, can we get rid of that `pyzmq` being written a second time?

Reply:
Nice catch, removed, thanks!